CHANGES.md

```diff
@@ -3,6 +3,9 @@ Version History
 ## 0.12.29 (03/26/2018)
  * Add new `COL_Unit.postCoursePrefix()` endpoint
 
+## 0.12.30 (03/30/2018)
```

Comment:
I can live with this being below 0.12.29 in this file, can you @applebya? it's the result of the merge

Reply:
Haha nah I can't, will fix now 😜

Reply:
good to go